easybuild/easyconfigs/l/LLVM/LLVM-7.0.0-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,48 @@
+#  jg (cscs)
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '7.0.0'
+
+homepage = "http://llvm.org/"
+
+description = """The LLVM Core libraries provide a modern source- and
+target-independent optimizer, along with code generation support for many
+popular CPUs (as well as some less common ones!) These libraries are built
+around a well specified code representation known as the LLVM intermediate
+representation ("LLVM IR"). The LLVM Core libraries are well documented, and it
+is particularly easy to invent your own language (or port an existing compiler)
+to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'dynamic': True}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = ["llvm-%(version)s.src.tar.xz"]
+
+#python = 'python'
```

Comment:
Do you need to keep all these comments?

Reply:
97d69172156481cffecfaa0476aab0053222e7f0